### PR TITLE
Update the importing logic for cpuinfo and psutil.

### PR DESCRIPTION
**Pull request by qlzh727**
Those two libs are usually not installed by default, and we should
not force people to install them if they just want to run resnet.

### CODE REVIEW

File: official/utils/logging/logger.py [hunk 1]
```diff
@@ -149,6 +142,10 @@ def _collect_cpu_info(run_info):
 
   cpu_info["num_cores"] = multiprocessing.cpu_count()
 
+  # Note: cpuinfo is not installed in the TensorFlow OSS tree.
+  # It is installable via pip.
+  import cpuinfo
+
```
Comment (karmel) [inline official/utils/logging/logger.py]: You will probably need a `  # pylint: disable=g-import-not-at-top` to make glint clear these.

File: official/utils/logging/logger.py [hunk 2]
```diff
@@ -149,6 +142,10 @@ def _collect_cpu_info(run_info):
 
   cpu_info["num_cores"] = multiprocessing.cpu_count()
 
+  # Note: cpuinfo is not installed in the TensorFlow OSS tree.
+  # It is installable via pip.
+  import cpuinfo
```
Comment (karmel) [inline official/utils/logging/logger.py]: You will probably need a `  # pylint: disable=g-import-not-at-top` to make glint clear these.

File: official/utils/logging/logger.py (hunk 1 above)
Comment (qlzh727) [reply official/utils/logging/logger.py]: Done.

File: official/utils/logging/logger.py (hunk 2 above)
Comment (qlzh727) [reply official/utils/logging/logger.py]: Done.

**Pull request merged**